lib/pf/services/manager/winbindd_child.pm

```diff
@@ -114,6 +115,34 @@ sub pidFile {
     }
 }
 
+=head2 _setupWatchForPidCreate
+
+This setups a watch on the run directory to wait for the pid to
```

Comment:
*This setups a watch on the run directory and its childs to wait for the pid to appear